features/conference/components/Conference.js

```diff
@@ -18,13 +27,25 @@ class Conference extends Component {
      */
     render() {
         return (
-            <ConferenceContainer>
+            <ConferenceContainer onPress = { this.onPress.bind(this) }>
                 <LargeVideo/>
-                <Toolbar navigator = { this.props.navigator }/>
-                <FilmStrip/>
+                <Toolbar
```

Comment:
Is there is a real need to have toolbar hidden instead of just removing it from DOM?
For me code like
```javascript
{ this.state.isToolbarVisible &&
    <Toolbar navigator = { this.props.navigator }/>}
```
seems to be more clear.
You may object that it will re-render then, but with React it's pretty fast so I think we can ignore it.